sql/plan.go

```diff
@@ -28,10 +29,11 @@ import (
 // planner is the centerpiece of SQL statement execution combining session
 // state and database state with the logic for SQL execution.
 type planner struct {
-	txn     *client.Txn
-	session Session
-	user    string
-	evalCtx parser.EvalContext
+	txn          *client.Txn
+	session      Session
+	user         string
+	evalCtx      parser.EvalContext
+	systemConfig *config.SystemConfig
```

Comment:
There isn't much point to using a pointer here. `config.SystemConfig` is immutable. The pointer will actually be slower too as it will require another dereference to get to the values. That really isn't a concern, but just pointing out that a value would be strictly better.